main.tf

```diff
@@ -1,8 +1,8 @@
 locals {
   create_random_password = var.create_db_instance && var.create_random_password && var.snapshot_identifier == null
   master_password        = try(random_password.master_password[0].result, var.password)
-  db_subnet_group_name   = !var.cross_region_replica && var.replicate_source_db != null ? null : try(module.db_subnet_group.db_subnet_group_id, var.db_subnet_group_name)
 
+  db_subnet_group_name    = var.create_db_subnet_group ? module.db_subnet_group.db_subnet_group_id : var.db_subnet_group_name
```

Comment:
Are you able to add this to the change log for version 4.1.0? This solved an issue for me, and it would have saved a bit of time if it was documented a bit more :D Thank you!

Reply:
Same exact thing happened to me.